bin/mixlib-install

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require_relative "../lib/mixlib/install/cli"
```

Comment:
We can avoid `require_relative` and more properly set up the environment with the following (this is [what we do in Omnibus](https://github.com/chef/omnibus/blob/master/bin/omnibus)):

```
# Trap interrupts to quit cleanly. See
# https://twitter.com/mitchellh/status/283014103189053442
Signal.trap("INT") { exit 1 }

$:.push File.expand_path("../../lib", __FILE__)
$stdout.sync = true
```